lua/terrortown/lang/en.lua

```diff
@@ -2129,6 +2127,14 @@ L.equipmenteditor_desc_builtin = "This equipment is built-in, it comes with TTT2
 L.help_roles_builtin = "This role is built-in, it comes with TTT2!"
 L.header_equipment_info = "Equipment information"
 
+-- 2023-12-20
+L.header_roles_magnetostick = "Magneto Stick"
+L.label_roles_ragdoll_pinning = "Enable ragdoll pinning"
+L.magneto_stick_help_carry_rag_pin = "Pin ragdoll"
+L.magneto_stick_help_carry_rag_drop = "Drop ragdoll"
+L.magneto_stick_help_carry_prop_release = "Release prop"
+L.magneto_stick_help_carry_prop_drop = "Drop prop"
```

Comment:
@TimGoll can you run the language tool here?

Reply:
no longer needed, so I will resolve this